WTWSMS/common/traits/02_traits.txt

```diff
@@ -1093,10 +1095,15 @@ schemer = {
 theologian = {
 	lifestyle = yes
 	
-	learning = 2
-	intrigue = 1
+	learning = 4
+	diplomacy = 2
 	
 	ruler_designer_cost = 10
+	church_opinion = 15
+
+	opposites = {
+		cynical
+	}
```

Comment:
You are correct, but because of the Events associated with cynicism are far more broad than theology. Cynicism doesn't mean Atheism, but the description does say he's close to being an unbeliever.